README.md

```diff
@@ -14,47 +14,63 @@
 [goreport-image]: https://goreportcard.com/badge/github.com/ubuntu/authd
 [goreport-url]: https://goreportcard.com/report/github.com/ubuntu/authd
 
+[docs-image]: https://readthedocs.com/projects/canonical-authd/badge/?version=latest
+[docs-url]: https://canonical-authd.readthedocs-hosted.com/en/latest/
+
 [![Code quality][actions-image]][actions-url]
 [![License][license-image]](COPYING)
 [![Code coverage][codecov-image]][codecov-url]
 [![Go Report Card][goreport-image]][goreport-url]
 [![Reference documentation][reference-documentation-image]][reference-documentation-url]
 
-Authd is an authentication daemon for cloud-based identity providers. It helps
-ensure the secure management of identity and access for Ubuntu machines
-anywhere in the world, on desktop and the server. Authd's modular design makes
-it a versatile authentication service that can integrate with multiple identity
-providers. MS Entra ID is currently supported and several other
-identity providers are under active development.
+[![Documentation Status][docs-image]][docs-url]
 
-## Documentation
+authd is an authentication daemon for cloud-based identity providers. It helps
+ensure the secure management of identity and access for Ubuntu machines anywhere
+in the world, on desktop and the server. authd's modular design makes it a
+versatile authentication service that can integrate with multiple identity
+providers. MS Entra ID is currently supported and several other identity
+providers are under active development.
 
-Documentation for authd is currently available as a [wiki](https://github.com/ubuntu/authd/wiki/01---Get-started-with-authd) that includes how-to guides on:
+## Documentation
 
-- [Installation](https://github.com/ubuntu/authd/wiki/02---How%E2%80%90to-install)
-- [Configuration](https://github.com/ubuntu/authd/wiki/03---How%E2%80%90to-configure)
-- Login through [GDM](https://github.com/ubuntu/authd/wiki/04---How%E2%80%90to-log-in-with-GDM) and [SSH](https://github.com/ubuntu/authd/wiki/05--How%E2%80%90to-log-in-over-SSH)
+If you want to know more about using authd, refer to the
+[official authd documentation](docs-url).
```

Comment:
I think the right syntax is:

```suggestion
[official authd documentation][docs-url].
```

:smile:

Reply:
OK. The square brackets work also and that was the syntax used in two guides I read. I will change ASAP!